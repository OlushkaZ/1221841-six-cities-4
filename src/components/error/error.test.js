import React from "react";
import renderer from "react-test-renderer";
import ErrorComponent from "./error.jsx";

it(`ReviewItem render`, () => {
  const tree = renderer
  .create(
      <ErrorComponent />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
