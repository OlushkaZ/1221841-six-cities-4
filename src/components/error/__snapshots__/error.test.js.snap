// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem render 1`] = `
<div
  className="main"
>
  <img
    src="https://i1.wp.com/neosmart.net/wiki/wp-content/uploads/sites/5/2015/07/0x00000109.jpg?ssl=1"
    style={
      Object {
        "height": "100vh",
        "objectFit": "cover",
        "width": "100vw",
      }
    }
  />
</div>
`;
